File: Estrutura_Condicional/Python/Ex2670.py
def tempo_minimo(A1, A2, A3):
    tempo_no_primeiro_andar = A2 * 2 + A3 * 4
    tempo_no_segundo_andar = A1 * 2 + A3 * 2
    tempo_no_terceiro_andar = A1 * 4 + A2 * 2

    return min(tempo_no_primeiro_andar, tempo_no_segundo_andar, tempo_no_terceiro_andar)


A1 = int(input())
A2 = int(input())
A3 = int(input())

print(tempo_minimo(A1, A2, A3))

File: Estrutura_Condicional/JavaScript/Ex2670.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift();};

function tempo_minimo(A1, A2, A3) {
    var tempo_no_primeiro_andar = A2 * 2 + A3 * 4;
    var tempo_no_segundo_andar = A1 * 2 + A3 * 2;
    var tempo_no_terceiro_andar = A1 * 4 + A2 * 2;

    return Math.min(tempo_no_primeiro_andar, tempo_no_segundo_andar, tempo_no_terceiro_andar);
}

var A1 = parseInt(prompt());
var A2 = parseInt(prompt());
var A3 = parseInt(prompt());

console.log(tempo_minimo(A1, A2, A3));